templates/role.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "betydb.fullname" . }}
+  labels:
+    {{- include "betydb.labels" . | nindent 4 }}
+rules:
+- apiGroups: [""]
+  resources: ["pods", "endpoints", "Services", "configmaps"]
```

Comment:
should this be
```suggestion
  resources: ["pods", "endpoints", "services", "configmaps"]
```

Reply:
Removed extra permissions from bety hook role